src/transformers/modeling_tf_utils.py

```diff
@@ -365,9 +367,7 @@ def input_processing(func, config, input_ids, **kwargs):
                 if tensor_name in parameter_names:
                     output[tensor_name] = input
                 else:
-                    raise ValueError(
```

Comment:
from the comment that was added above it seems like the `tensor_name` has to be in `parameter_names` no? But we still allow tensors without any parameter name?

Reply:
should we then maybe at least raise a warning here at the moment? And extend the functions docstring with a sentence that there might be unexpected behavior if the inputs have no names?

Reply:
As you can see in the example, the tensors during explicite model creation cannot have a name that belongs to any parameter name. So yes we allow tensors that have a different name than the parameters. But, the input names must have a name that belongs to the parameters in order to be able to have a proper order in the list. It is more obvious when reading the full example above.

Reply:
@patrickvonplaten I have updated the comment in order to make it clearer. Does-it sounds more understandable for you?